### ISSUE TITLE
Crash on Linux if not fullscreen

### ISSUE THREAD

**Issue by AndreaMonzini** [wjakob/instant-meshes#77]
Hello,

i'm testing instant-meshes on Linux and it crash just after the run.
This is the terminal report:
```
[linux@localhost instant-meshes-linux]$ ./'Instant Meshes'
Running in GUI mode, start with -h for instructions on batch mode.
Compiling shaders .. done. (took 2.0ms)
Unable to load image data: Could not open image directory!
```
But if i run it in full screen using `./Instant Meshes-F`  it works !

My system:
Ryzen 1700, AMD RX 580 ( Mesa driver), 32 GB ram, Fedora 28, Kernel 4.16.8 ( fresh install ).

**Comment by Houkime** [comment]
Same here.

Ryzen 1300, AMD RX 550 (Mesa), 8Gb ram, Arch Linux, Linux kernel  5.2.11

```
Running in GUI mode, start with -h for instructions on batch mode.
Compiling shaders .. done. (took 1.0ms)
Unable to load image data: Could not open image directory!
Killed


```

**Comment by thima2017** [comment]
I can confirm it. It doesn't crash in my case it just freezes and shows nothing in the opened window. Ubuntu 18.04, AMD RX 480.
[strace.txt](https://github.com/wjakob/instant-meshes/files/3801561/strace.txt)

**Comment by rviertel** [comment]
For me it is non-responsive when it launches. -F does not fix it for me, but if I launch with sudo it intermittently will work correctly. Running on a Ryzen 5 3600 and AMD RX 570 GPU

**Comment by arun54321** [comment]
> ./Instant Meshes -F it works !

Yes this works. but on wayland it cannot open import box properly. Works fine on X11.

**Comment by finnbainbridge** [comment]
I've had the same problem where it fails to launch, fullscreen or not fullscreen. Fortunately, it works perfectly through wine.

**Comment by mahaidong** [comment]
none of this two working for me. fail too

**Comment by matteounimib** [comment]
I have the same problem.
I tried both downaded executable and compiled version.
Running with -F flag does not solve the issue.

> Running in GUI mode, start with -h for instructions on batch mode.
> Detected Intel HD Graphics card, disabling MSAA as a precaution ..
> Compiling shaders .. done. (took 7.0ms)
> Unable to load image data: Could not open image directory!
> X Error of failed request:  BadMatch (invalid parameter attributes)
>  Major opcode of failed request:  42 (X_SetInputFocus)
>  Serial number of failed request:  214
>  Current serial number in output stream:  216
